lint/vale.bzl

```diff
@@ -82,9 +82,12 @@ def vale_action(ctx, executable, srcs, styles, config, stdout, exit_code = None)
         stdout: output file containing stdout of Vale
         exit_code: output file containing Vale exit code.
             If None, then fail the build when Vale exits non-zero.
+        format: the value for the --output CLI flag
```

Comment:
nit: mention default is "CLI" and which is stylish

Reply:
stardoc picks up the default value, it's already documented in the markdown table

Reply:
renamed to `output` to match the flag